week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -11,12 +11,15 @@ We made a start on this for you here:
 let bottle = {
   volume: 0,
   fill: function() {
+    this.volume = 100;
     // calling this function should make you bottles volume = 100;
   },
   drink: function() {
+    this.volume -= 10;
     // calling this function should decrease your bottles volume by 10;
   },
   empty: function() {
+    return this.volume === 0 ?  true :  false

```

Comment:
Great work! 🙌 
Tip: To keep it shorter you could just do
`return this.volume === 0` which evaluates to the same